app/controllers/users_controller.rb

```diff
@@ -18,10 +26,47 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
```

Comment:
What will happen if it don't find the user? How you can handle this situation(if @user is nil)

Reply:
I already handle this case.
Before you go to the edit page, you have to log in first.
I defined a method called logged_in_user in users controller.
So basically, if an id not exist, you can't log in with that id and you can't edit or update.